lib/sequenceserver/job_remover.rb

```diff
@@ -0,0 +1,59 @@
+require 'sequenceserver/job.rb'
+
+module SequenceServer 
+  class JobRemover
+
+    DEFAULT_LIFETIME = 86400
+
+    def initialize(lifetime)
+      parse lifetime
+      spawn_cleanup if @clean_needed
+    end
+
+    def spawn_cleanup
+      Thread.abort_on_exception = true if ENV['RACK_ENV'] == 'development'
```

Comment:
This a good measure. However, this changes the behaviour of all threads that are spawned by SequenceServer, including worker threads that run BLAST. Thus this should be configured during program initialisation.

Also, what happens if an exception is raised in this thread in production? The thread and thus job cleaner will terminate _silently_. I think it's better that you rescue all exceptions inside the loop, log them, and continue with the next tick.